lib/rubocop/cop/lint/duplicated_key.rb

```diff
@@ -0,0 +1,37 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for duplicated keys in hash literals.
+      #
+      # This cop mirrors a warning in Ruby 2.2.
+      #
+      # @example
+      #   hash = { food: 'apple', food: 'orange' }
+      class DuplicatedKey < Cop
+        MSG = 'Duplicated key in hash literal'
+
+        LITERALS = [:sym, :str, :float, :int]
+
+        def on_hash(node)
+          keys = []
+
+          hash_pairs = *node
+          hash_pairs.each do |pair|
```

Comment:
this might help understand:
```
[19] pry(main)> a, b = Parser::CurrentRuby.parse('[1,2]'); p a, b
(array
  (int 1)
  (int 2))
nil
[20] pry(main)> a, b = *Parser::CurrentRuby.parse('[1,2]'); p a, b
(int 1)
(int 2)
```

naturally, there's no explicit right-hand side in this assignment.